build.gradle

```diff
@@ -25,7 +25,6 @@ allprojects {
         maven { url "https://repo.maven.apache.org/maven2" }
         maven { url "http://cloudant.github.io/cloudant-sync-eap/repository" }
         maven { url "https://s3.amazonaws.com/repo.commonsware.com" }
-        maven { url 'https://dl.bintray.com/ibm-watson-health/ibm-fhir-server-releases'}
```

Comment:
Have you confirmed this fix is correct? I mean it might not cause issues with other libraries?

Reply:
@hamza-vd yes. Allan helped to confirm this. "Hi Roy. After the sunset of Bintray  services, we no longer have dependencies resolving from Bintray. That line should be deleted from where the repositories are declared. The IBM-FHIR dependency should resolve from jcenter()"